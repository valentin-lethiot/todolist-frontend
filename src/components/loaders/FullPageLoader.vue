<script setup lang="ts">
import Loader from '@/components/loaders/Loader.vue'
</script>

<template>
  <div class="flex justify-center items-center h-screen w-screen">
    <Loader />
  </div>
</template>
