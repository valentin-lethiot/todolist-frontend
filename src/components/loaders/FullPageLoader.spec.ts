import { describe, beforeEach, expect, test } from 'vitest'
import FullPageLoader from './FullPageLoader.vue'
import { mount, VueWrapper } from '@vue/test-utils'

describe('FullPageLoader', () => {
  let wrapper: VueWrapper

  beforeEach(() => {
    wrapper = mount(FullPageLoader)
  })

  describe('render', () => {
    test('should render the full page loader', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
