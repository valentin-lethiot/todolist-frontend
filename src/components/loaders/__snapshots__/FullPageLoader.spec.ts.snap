// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FullPageLoader > render > should render the full page loader 1`] = `
"<div class="flex justify-center items-center h-screen w-screen">
  <div style="height: 200px; width: 200px;"><canvas style="height: 100%; width: 100%;"></canvas></div>
</div>"
`;
